contracts/test/_global-hooks.js

```diff
@@ -34,11 +39,26 @@ mocha.before(function () {
 
   // If you are running unit tests, scrape out all fork tests.
   // For fork tests, scrape out all unit tests.
-  root.suites = root.suites.filter(
-    (s) =>
-      s.file.endsWith(".fork-test.js") == isForkTest &&
-      s.file.endsWith(".arb.fork-test.js") == isArbFork
-  );
+  root.suites = root.suites.filter((s) => {
+    const isMainnetForkTestFile = s.file.endsWith("mainnet.fork-test.js");
+    const isHoleskyTestFile = s.file.endsWith(".holesky.fork-test.js");
+    const isArbTestFile = s.file.endsWith(".arb.fork-test.js");
+    const isBaseTestFile = s.file.endsWith(".base.fork-test.js");
```

Comment:
One of the reasons I suggested this pattern is that we can parse the network from the file name and check it against `FORK_NETWORK_NAME` to simplify this a lot